src/CarbonAware.Aggregators/src/CarbonAware/CarbonAwareAggregator.cs

```diff
@@ -61,9 +61,9 @@ public async Task<IEnumerable<EmissionsForecast>> GetCurrentForecastDataAsync(ID
                 forecast.EndTime = end;
                 forecast.ForecastData = IntervalHelper.FilterByDuration(forecast.ForecastData, start, end);
                 forecast.ForecastData = forecast.ForecastData.RollingAverage(windowSize);
+                forecast.OptimalDataPoint = GetOptimalEmissions(forecast.ForecastData);
```

Comment:
Why not leave this inside the if statement? You do the `forecast.ForecastData.Any()` check in the` GetOptimalEmissions` then set to `null` anyways so by leaving in the `if` statement, wouldn't you just always only hit it if there is forecastdata, and by default that param would stay null if not?

Reply:
The reason we did this was to avoid confusion for future devs asking "why are we checking `data.Any()` and then rechecking it in the immediate next method?"

If people feel strongly that that tradeoff is not worth it, I don't feel super strongly about either choice, so we could revert.

Reply:
Did not feel strongly, just more wondering if I was missing some context :) This is fine.